src/components/Hero.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import skyline from '../images/skyline.svg'
+
+const Hero = () => {
+  return (
+    <section>
+      <div className="hero-content-container">
+        <h4>A CODE FOR AMERICA BRIGADE</h4>
```

Comment:
There's a lot of discussion on how to handle eyebrow text. There's no official way to handle it (some argue HTML does not modern designs). But we definitely can't have our `h` tags out of order. So here's how I think we should handle it;

Let's wrap lines 8 and 9 in an `h2` tag (most Hero's contain the page heading and that's what we'll assume is happening here). Then let's wrap "A CODE FOR AMERICA BRIGADE" in a `span` tag with a className of "eyebrow-text". 

And then one last semantic change on this line. Let's keep reusable and simply pass this content as a prop. 

So something like `eyebrowText`.